Source/Core/AudioCommon/OpenALStream.cpp

```diff
@@ -76,7 +76,7 @@ static bool InitLibrary()
 
   if (!InitFunctions())
   {
-    ::FreeLibrary(s_openal_dll);
```

Comment:
I don't know if there is a reason that `FreeLibrary` is explicitly requested from the global namespace here, but all of our other `FreeLibrary` usages also do this. Is there a name conflict on specific targets or something?